src/Meilisearch/Query/SubQueriesBuilderInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\Meilisearch\Query;
+
+use Meilisearch\Contracts\SearchQuery;
+use Setono\SyliusMeilisearchPlugin\Document\Metadata\Facet;
+
+interface SubQueriesBuilderInterface
+{
+    /**
+     * @param array<string, Facet> $facets
+     * @param array<string> $facetsNames
+     *
+     * @return array<SearchQuery>
+     */
+    public function build(
```

Comment:
Do we need to both input `$facets` _and_ `$facetsNames`?

Reply:
In fact we don't 👍